docs/features/video-protection.md

```diff
@@ -0,0 +1,108 @@
+# Video Protection
+
+Videos can be protected in JW platform in two ways:
+
+- **Signed URLs:** A player can only **access** video URLs from JW Player backend using a time-bound JWT token
+- **DRM:** A player can only **play** videos using a time-bound decryption key. 
+
+The  JW Web Player, as well as the other SDKs, supports these mechanismns out-the-box. However this require a server side entitlement service which is **NOT** part of the web app and needs to be **custom developed**.
```

Comment:
I think we can call this an authorization service, or subscription service, or something like that? Unless entitlements is a term common with broadcasters?

Reply:
Authorization service is indeed a more proper term.  Cleeng is actually the entitlement service.